app/src/main/java/com/eventyay/organizer/utils/Utils.java

```diff
@@ -1,12 +1,16 @@
 package com.eventyay.organizer.utils;
 
+import android.util.DisplayMetrics;
+
 import com.eventyay.organizer.data.event.Event;
 
 import java.util.List;
 
 import io.reactivex.Observable;
 import io.reactivex.Single;
 
+import static com.raizlabs.android.dbflow.config.FlowManager.getContext;
+
```

Comment:
Unneeded

Reply:
The activity context is required in the dpToPx function to get an instance of the display metrics

Reply:
@iamareebjamal should I pass the context as a parameter in that function ?